homeassistant/components/zwave_js/fan.py

```diff
@@ -85,7 +84,9 @@ def __init__(
     ) -> None:
         """Initialize the fan."""
         super().__init__(config_entry, driver, info)
-        self._target_value = self.get_zwave_value(TARGET_VALUE_PROPERTY)
+        if not (target_value := self.get_zwave_value(TARGET_VALUE_PROPERTY)):
+            raise HomeAssistantError("Missing target value on device.")
```

Comment:
Maybe we should guard this already in discovery info? If the entity can't be added, the discovery isn't useful.

Reply:
yeah probably a good idea, we can then assert instead of raising an exception that we should never actually hit